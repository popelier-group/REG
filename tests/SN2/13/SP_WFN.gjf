# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.80432400   -0.00005200    0.00020800
 H                 -0.87632800   -0.90866000   -0.56611000
 H                 -0.87633100    0.94471500   -0.50347500
 H                 -0.87628400   -0.03621700    1.07025600
 Br                 1.61188400    0.00000400   -0.00001900
 Cl                -3.28594700    0.00001300   -0.00004900

frame.wfn
