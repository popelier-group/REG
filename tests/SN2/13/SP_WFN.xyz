
6

C        0.000145   -0.006325    0.691180
H       -0.574402   -0.910372    0.754935
H       -0.494950    0.942282    0.771759
H        1.069814   -0.052851    0.762807
Br        0.000114    0.015661   -1.724928
Cl       -0.000313   -0.028779    3.172701
