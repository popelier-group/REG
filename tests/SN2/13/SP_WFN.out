 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.13/Gau-138601.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.13/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    138603.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.80432  -0.00005   0.00021 
 H                    -0.87633  -0.90866  -0.56611 
 H                    -0.87633   0.94472  -0.50348 
 H                    -0.87628  -0.03622   1.07026 
 Br                    1.61188   0.       -0.00002 
 Cl                   -3.28595   0.00001  -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.804324   -0.000052    0.000208
      2          1           0       -0.876328   -0.908660   -0.566110
      3          1           0       -0.876331    0.944715   -0.503475
      4          1           0       -0.876284   -0.036217    1.070256
      5         35           0        1.611884    0.000004   -0.000019
      6         17           0       -3.285947    0.000013   -0.000049
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.073065   0.000000
     3  H    1.073064   1.854433   0.000000
     4  H    1.073074   1.854414   1.854416   0.000000
     5  Br   2.416208   2.708750   2.708719   2.708834   0.000000
     6  Cl   2.481623   2.636736   2.636693   2.636919   4.897831
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000145   -0.006325    0.691180
      2          1           0       -0.574402   -0.910372    0.754935
      3          1           0       -0.494950    0.942282    0.771759
      4          1           0        1.069814   -0.052851    0.762807
      5         35           0        0.000114    0.015661   -1.724928
      6         17           0       -0.000313   -0.028779    3.172701
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         145.8194242           0.8501830           0.8501830
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       172.5087984043 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  6.97D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.38730399     A.U. after   12 cycles
            NFock= 12  Conv=0.25D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.62922-101.30698 -62.28222 -56.10125 -56.09932
 Alpha  occ. eigenvalues --  -56.09932 -10.09000  -9.22023  -8.49110  -6.98066
 Alpha  occ. eigenvalues --   -6.97736  -6.97736  -6.31807  -6.31179  -6.31179
 Alpha  occ. eigenvalues --   -2.42966  -2.42794  -2.42794  -2.42298  -2.42298
 Alpha  occ. eigenvalues --   -0.59887  -0.56463  -0.52451  -0.30753  -0.30753
 Alpha  occ. eigenvalues --   -0.17636  -0.11434  -0.11434  -0.09923  -0.09923
 Alpha  occ. eigenvalues --   -0.09369
 Alpha virt. eigenvalues --    0.09190   0.11456   0.15034   0.15034   0.15709
 Alpha virt. eigenvalues --    0.17852   0.18414   0.18415   0.20360   0.21684
 Alpha virt. eigenvalues --    0.21684   0.23282   0.23282   0.24727   0.30562
 Alpha virt. eigenvalues --    0.31704   0.31705   0.36362   0.39147   0.53159
 Alpha virt. eigenvalues --    0.54531   0.54531   0.56783   0.56783   0.60835
 Alpha virt. eigenvalues --    0.64715   0.64716   0.69920   0.76342   0.76592
 Alpha virt. eigenvalues --    0.76593   0.81993   0.81993   0.83814   0.83815
 Alpha virt. eigenvalues --    0.85566   0.89680   1.10165   1.10166   1.10513
 Alpha virt. eigenvalues --    1.10514   1.12516   1.16487   1.22537   1.22538
 Alpha virt. eigenvalues --    1.33965   1.52692   1.62132   1.62133   1.67866
 Alpha virt. eigenvalues --    1.96749   2.00644   2.00645   2.11860   2.11861
 Alpha virt. eigenvalues --    2.19148   2.19148   2.30684   2.48325   2.48326
 Alpha virt. eigenvalues --    2.56838   2.61815   2.61816   2.68017   2.77340
 Alpha virt. eigenvalues --    2.77340   2.98865   3.22272   3.86372   3.86373
 Alpha virt. eigenvalues --    4.33030   4.33031   4.35809   4.35809   4.54980
 Alpha virt. eigenvalues --    6.75067   7.73902   7.73904   7.82460   9.86852
 Alpha virt. eigenvalues --   24.03978  26.04606  26.04607  26.25725  48.05221
 Alpha virt. eigenvalues --  215.84325 289.94277 289.94277 290.027141020.82255
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.041754   0.418730   0.418735   0.418712   0.050689  -0.045442
     2  H    0.418730   0.514795  -0.032014  -0.032025  -0.036944  -0.015340
     3  H    0.418735  -0.032014   0.514793  -0.032023  -0.036949  -0.015343
     4  H    0.418712  -0.032025  -0.032023   0.514804  -0.036925  -0.015332
     5  Br   0.050689  -0.036944  -0.036949  -0.036925  35.641775   0.008074
     6  Cl  -0.045442  -0.015340  -0.015343  -0.015332   0.008074  17.738875
 Mulliken charges:
               1
     1  C   -0.303178
     2  H    0.182799
     3  H    0.182801
     4  H    0.182789
     5  Br  -0.589718
     6  Cl  -0.655492
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.245211
     5  Br  -0.589718
     6  Cl  -0.655492
 Electronic spatial extent (au):  <R**2>=           1143.0091
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0009    Y=              0.0437    Z=             -4.8254  Tot=              4.8256
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.8066   YY=            -43.8101   ZZ=            -88.8473
   XY=             -0.0000   XZ=              0.0044   YZ=              0.4086
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             15.0148   YY=             15.0112   ZZ=            -30.0260
   XY=             -0.0000   XZ=              0.0044   YZ=              0.4086
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.0891  YYY=              0.6063  ZZZ=           -156.7468  XYY=             -1.0828
  XXY=              0.0151  XXZ=            -17.1219  XZZ=              0.0108  YZZ=              1.1116
  YYZ=            -17.1280  XYZ=             -0.0099
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -70.7760 YYYY=            -70.8935 ZZZZ=          -1789.3986 XXXY=             -0.0077
 XXXZ=              0.8357 YYYX=              0.0208 YYYZ=              6.3578 ZZZX=              0.0908
 ZZZY=              9.3373 XXYY=            -23.6096 XXZZ=           -253.4452 YYZZ=           -253.4884
 XXYZ=              1.9853 YYXZ=             -0.7567 ZZXY=             -0.0146
 N-N= 1.725087984043D+02 E-N=-7.697519615455D+03  KE= 3.069963544139D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE763\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.804324,-0.000052,0.000208\H,0,-0.876328,-0.90866,-0.56611\H,0,-
 0.876331,0.944715,-0.503475\H,0,-0.876284,-0.036217,1.070256\Br,0,1.61
 1884,0.000004,-0.000019\Cl,0,-3.285947,0.000013,-0.000049\\Version=ES6
 4L-G16RevC.01\State=1-A\HF=-3074.387304\RMSD=2.548e-09\Dipole=1.898519
 2,-0.0000496,0.0002024\Quadrupole=-22.326333,11.1632272,11.1631057,0.0
 001641,-0.0005802,-0.0000051\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 FAULTILY FAULTLESS, ICILY REGULAR, SPLENDIDLY NULL...
           MAUDE BY TENNYSON
 Job cpu time:       0 days  0 hours  0 minutes 17.3 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 10.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:03 2020.
